scripts/install.sh

```diff
@@ -229,6 +229,7 @@ function main() {
     tasks
     preflights
     prompts
+    setup_installer_crd
```

Comment:
From talking to marc this looks like it may be a good place to add this, but want to hear feedback

Reply:
kubectl doesn't exist at this point on new installs

Reply:
edited